tools/tsp-client/src/index.ts

```diff
@@ -355,6 +362,16 @@ async function main() {
           }
         }
         break;
+    case "sortSwagger":
+        Logger.info("Sorting a swagger content...");
+        let swaggerFile =  options.swaggerFile;
+        if (swaggerFile === undefined || await !doesFileExist(swaggerFile)) {
+            throw new Error(`Swagger file not found: ${swaggerFile??"[Not Specified]"}`);
+        }
+        swaggerFile = normalizePath(resolve(swaggerFile));
```

Comment:
Should we be doing this before we do the doesFileExist check on line 368?

Reply:
Seeing the above code, it was done this way. I am not sure if this does anything as if `doesFileExist` works with the path, `readFile` should work as well. But I can adjust.